· Creating environments
· Discovering benchmarks
·· Uninstalling from conda-py3.6-dask-distributed-gcsfs-h5netcdf-h5py-lz4-netcdf4-numpy-pip+dask_kubernetes-pip+h5pyd-pyyaml-rasterio-xarray-zarrPrevious results JSON found. Moving to backup file
Traceback (most recent call last):
  File "run_benchmarks.py", line 292, in <module>
    main()
  File "run_benchmarks.py", line 277, in main
    call(['asv', 'run', '-e', '-b', args.benchmark[0]])
  File "/opt/conda/lib/python3.6/subprocess.py", line 269, in call
    return p.wait(timeout=timeout)
  File "/opt/conda/lib/python3.6/subprocess.py", line 1457, in wait
    (pid, sts) = self._try_wait(0)
  File "/opt/conda/lib/python3.6/subprocess.py", line 1404, in _try_wait
    (pid, sts) = os.waitpid(self.pid, wait_flags)
KeyboardInterrupt
· Creating environments
· Discovering benchmarks
·· Uninstalling from conda-py3.6-dask-distributed-gcsfs-h5netcdf-h5py-lz4-netcdf4-numpy-pip+dask_kubernetes-pip+h5pyd-pyyaml-rasterio-xarray-zarr
·· Building for conda-py3.6-dask-distributed-gcsfs-h5netcdf-h5py-lz4-netcdf4-numpy-pip+dask_kubernetes-pip+h5pyd-pyyaml-rasterio-xarray-zarr
·· Installing into conda-py3.6-dask-distributed-gcsfs-h5netcdf-h5py-lz4-netcdf4-numpy-pip+dask_kubernetes-pip+h5pyd-pyyaml-rasterio-xarray-zarr
· Running 4 total benchmarks (1 commits * 1 environments * 4 benchmarks)
[  0.00%] · For pangeo-storage-benchmarks commit hash db86254c:
[  0.00%] ·· Building for conda-py3.6-dask-distributed-gcsfs-h5netcdf-h5py-lz4-netcdf4-numpy-pip+dask_kubernetes-pip+h5pyd-pyyaml-rasterio-xarray-zarr
[  0.00%] ·· Benchmarking conda-py3.6-dask-distributed-gcsfs-h5netcdf-h5py-lz4-netcdf4-numpy-pip+dask_kubernetes-pip+h5pyd-pyyaml-rasterio-xarray-zarr
[ 25.00%] ··· Running gcp_kubernetes_read_netCDF.llc4320_benchmarks.time_read                                                                                                                                                                                                        ok
[ 25.00%] ···· 
               ========= ==============
               --        z_chunksize / n_workers / run_num
               --------- --------------
                backend   10 / 100 / 1 
               ========= ==============
                  FUSE       5.80m     
               ========= ==============

[ 25.00%] ····· 
                
                For parameters: 'FUSE', 10, 100, 1
                /home/jovyan/dev/storage-benchmarks/.asv/env/4147cb8bacd14ebb1c15e8248dc80f49/lib/python3.6/importlib/_bootstrap.py:219: RuntimeWarning: numpy.dtype size changed, may indicate binary incompatibility. Expected 96, got 88
                  return f(*args, **kwds)

[ 50.00%] ··· Running gcp_kubernetes_read_netCDF.llc4320_ds_size.track_megabytes                                                                                                                                                                                     1111653.8307113647
[ 50.00%] ····· /home/jovyan/dev/storage-benchmarks/.asv/env/4147cb8bacd14ebb1c15e8248dc80f49/lib/python3.6/importlib/_bootstrap.py:219: RuntimeWarning: numpy.dtype size changed, may indicate binary incompatibility. Expected 96, got 88
                return f(*args, **kwds)

[ 75.00%] ··· Running gcp_kubernetes_read_zarr.llc4320_benchmarks.time_read                                                                                                                                                                                                          ok
[ 75.00%] ···· 
               ========= ============ ============ ============= ============= ============= =============
               --                                z_chunksize / n_workers / run_num                        
               --------- ---------------------------------------------------------------------------------
                backend   1 / 60 / 1   1 / 80 / 1   1 / 100 / 1   1 / 120 / 1   1 / 140 / 1   1 / 160 / 1 
               ========= ============ ============ ============= ============= ============= =============
                  GCS       5.32m        3.47m         2.91m         2.41m         2.08m         1.98m    
               ========= ============ ============ ============= ============= ============= =============

[ 75.00%] ····· 
                
                For parameters: 'GCS', 1, 60, 1
                /home/jovyan/dev/storage-benchmarks/.asv/env/4147cb8bacd14ebb1c15e8248dc80f49/lib/python3.6/importlib/_bootstrap.py:219: RuntimeWarning: numpy.dtype size changed, may indicate binary incompatibility. Expected 96, got 88
                  return f(*args, **kwds)
                
                For parameters: 'GCS', 1, 80, 1
                /home/jovyan/dev/storage-benchmarks/.asv/env/4147cb8bacd14ebb1c15e8248dc80f49/lib/python3.6/importlib/_bootstrap.py:219: RuntimeWarning: numpy.dtype size changed, may indicate binary incompatibility. Expected 96, got 88
                  return f(*args, **kwds)
                
                For parameters: 'GCS', 1, 100, 1
                /home/jovyan/dev/storage-benchmarks/.asv/env/4147cb8bacd14ebb1c15e8248dc80f49/lib/python3.6/importlib/_bootstrap.py:219: RuntimeWarning: numpy.dtype size changed, may indicate binary incompatibility. Expected 96, got 88
                  return f(*args, **kwds)
                
                For parameters: 'GCS', 1, 120, 1
                /home/jovyan/dev/storage-benchmarks/.asv/env/4147cb8bacd14ebb1c15e8248dc80f49/lib/python3.6/importlib/_bootstrap.py:219: RuntimeWarning: numpy.dtype size changed, may indicate binary incompatibility. Expected 96, got 88
                  return f(*args, **kwds)
                
                For parameters: 'GCS', 1, 140, 1
                /home/jovyan/dev/storage-benchmarks/.asv/env/4147cb8bacd14ebb1c15e8248dc80f49/lib/python3.6/importlib/_bootstrap.py:219: RuntimeWarning: numpy.dtype size changed, may indicate binary incompatibility. Expected 96, got 88
                  return f(*args, **kwds)
                
                For parameters: 'GCS', 1, 160, 1
                /home/jovyan/dev/storage-benchmarks/.asv/env/4147cb8bacd14ebb1c15e8248dc80f49/lib/python3.6/importlib/_bootstrap.py:219: RuntimeWarning: numpy.dtype size changed, may indicate binary incompatibility. Expected 96, got 88
                  return f(*args, **kwds)

[100.00%] ··· Running gcp_kubernetes_read_zarr.llc4320_ds_size.track_megabytes                                                                                                                                                                                       1111653.8307113647
[100.00%] ····· /home/jovyan/dev/storage-benchmarks/.asv/env/4147cb8bacd14ebb1c15e8248dc80f49/lib/python3.6/importlib/_bootstrap.py:219: RuntimeWarning: numpy.dtype size changed, may indicate binary incompatibility. Expected 96, got 88
                return f(*args, **kwds)

· Creating environments
· Discovering benchmarks
·· Uninstalling from conda-py3.6-dask-distributed-gcsfs-h5netcdf-h5py-lz4-netcdf4-numpy-pip+dask_kubernetes-pip+h5pyd-pyyaml-rasterio-xarray-zarr
·· Building for conda-py3.6-dask-distributed-gcsfs-h5netcdf-h5py-lz4-netcdf4-numpy-pip+dask_kubernetes-pip+h5pyd-pyyaml-rasterio-xarray-zarr
·· Installing into conda-py3.6-dask-distributed-gcsfs-h5netcdf-h5py-lz4-netcdf4-numpy-pip+dask_kubernetes-pip+h5pyd-pyyaml-rasterio-xarray-zarr
· Running 2 total benchmarks (1 commits * 1 environments * 2 benchmarks)
[  0.00%] · For pangeo-storage-benchmarks commit hash 75a490e9:
[  0.00%] ·· Building for conda-py3.6-dask-distributed-gcsfs-h5netcdf-h5py-lz4-netcdf4-numpy-pip+dask_kubernetes-pip+h5pyd-pyyaml-rasterio-xarray-zarr
[  0.00%] ·· Benchmarking conda-py3.6-dask-distributed-gcsfs-h5netcdf-h5py-lz4-netcdf4-numpy-pip+dask_kubernetes-pip+h5pyd-pyyaml-rasterio-xarray-zarr
[ 50.00%] ··· Running gcp_kubernetes_write_zarr.rand_ds_size.track_megabytes             68664.55078125
[ 50.00%] ····· /home/jovyan/dev/storage-benchmarks/.asv/env/4147cb8bacd14ebb1c15e8248dc80f49/lib/python3.6/importlib/_bootstrap.py:219: RuntimeWarning: numpy.dtype size changed, may indicate binary incompatibility. Expected 96, got 88
                return f(*args, **kwds)

[100.00%] ··· Running gcp_kubernetes_write_zarr.synthetic_benchmarks.time_synthetic_write   1/15 failed
[100.00%] ···· 
               ========= ============= ======== ======== ========= ========= =========
               --                                    n_workers / run_num              
               ----------------------- -----------------------------------------------
                backend   z_chunksize   60 / 1   80 / 1   100 / 1   120 / 1   140 / 1 
               ========= ============= ======== ======== ========= ========= =========
                  GCS          1        24.37s   20.41s    20.09s    15.99s    14.91s 
                  GCS          5        20.99s   31.42s    19.06s    failed    15.56s 
                  GCS          10       45.97s   23.69s    22.36s    20.41s    33.13s 
               ========= ============= ======== ======== ========= ========= =========

[100.00%] ····· 
                
                For parameters: 'GCS', 1, 60, 1
                /home/jovyan/dev/storage-benchmarks/.asv/env/4147cb8bacd14ebb1c15e8248dc80f49/lib/python3.6/importlib/_bootstrap.py:219: RuntimeWarning: numpy.dtype size changed, may indicate binary incompatibility. Expected 96, got 88
                  return f(*args, **kwds)
                
                For parameters: 'GCS', 1, 80, 1
                /home/jovyan/dev/storage-benchmarks/.asv/env/4147cb8bacd14ebb1c15e8248dc80f49/lib/python3.6/importlib/_bootstrap.py:219: RuntimeWarning: numpy.dtype size changed, may indicate binary incompatibility. Expected 96, got 88
                  return f(*args, **kwds)
                
                For parameters: 'GCS', 1, 100, 1
                /home/jovyan/dev/storage-benchmarks/.asv/env/4147cb8bacd14ebb1c15e8248dc80f49/lib/python3.6/importlib/_bootstrap.py:219: RuntimeWarning: numpy.dtype size changed, may indicate binary incompatibility. Expected 96, got 88
                  return f(*args, **kwds)
                
                For parameters: 'GCS', 1, 120, 1
                /home/jovyan/dev/storage-benchmarks/.asv/env/4147cb8bacd14ebb1c15e8248dc80f49/lib/python3.6/importlib/_bootstrap.py:219: RuntimeWarning: numpy.dtype size changed, may indicate binary incompatibility. Expected 96, got 88
                  return f(*args, **kwds)
                
                For parameters: 'GCS', 1, 140, 1
                /home/jovyan/dev/storage-benchmarks/.asv/env/4147cb8bacd14ebb1c15e8248dc80f49/lib/python3.6/importlib/_bootstrap.py:219: RuntimeWarning: numpy.dtype size changed, may indicate binary incompatibility. Expected 96, got 88
                  return f(*args, **kwds)
                
                For parameters: 'GCS', 5, 60, 1
                /home/jovyan/dev/storage-benchmarks/.asv/env/4147cb8bacd14ebb1c15e8248dc80f49/lib/python3.6/importlib/_bootstrap.py:219: RuntimeWarning: numpy.dtype size changed, may indicate binary incompatibility. Expected 96, got 88
                  return f(*args, **kwds)
                
                For parameters: 'GCS', 5, 80, 1
                /home/jovyan/dev/storage-benchmarks/.asv/env/4147cb8bacd14ebb1c15e8248dc80f49/lib/python3.6/importlib/_bootstrap.py:219: RuntimeWarning: numpy.dtype size changed, may indicate binary incompatibility. Expected 96, got 88
                  return f(*args, **kwds)
                
                For parameters: 'GCS', 5, 100, 1
                /home/jovyan/dev/storage-benchmarks/.asv/env/4147cb8bacd14ebb1c15e8248dc80f49/lib/python3.6/importlib/_bootstrap.py:219: RuntimeWarning: numpy.dtype size changed, may indicate binary incompatibility. Expected 96, got 88
                  return f(*args, **kwds)
                
                For parameters: 'GCS', 5, 120, 1
                /home/jovyan/dev/storage-benchmarks/.asv/env/4147cb8bacd14ebb1c15e8248dc80f49/lib/python3.6/importlib/_bootstrap.py:219: RuntimeWarning: numpy.dtype size changed, may indicate binary incompatibility. Expected 96, got 88
                  return f(*args, **kwds)
                Traceback (most recent call last):
                  File "/home/jovyan/.local/lib/python3.6/site-packages/asv/benchmark.py", line 795, in <module>
                    commands[mode](args)
                  File "/home/jovyan/.local/lib/python3.6/site-packages/asv/benchmark.py", line 772, in main_run
                    result = benchmark.do_run()
                  File "/home/jovyan/.local/lib/python3.6/site-packages/asv/benchmark.py", line 456, in do_run
                    return self.run(*self._current_params)
                  File "/home/jovyan/.local/lib/python3.6/site-packages/asv/benchmark.py", line 548, in run
                    all_runs.extend(timer.repeat(repeat, number))
                  File "/home/jovyan/dev/storage-benchmarks/.asv/env/4147cb8bacd14ebb1c15e8248dc80f49/lib/python3.6/timeit.py", line 206, in repeat
                    t = self.timeit(number)
                  File "/home/jovyan/dev/storage-benchmarks/.asv/env/4147cb8bacd14ebb1c15e8248dc80f49/lib/python3.6/timeit.py", line 178, in timeit
                    timing = self.inner(it, self.timer)
                  File "<timeit-src>", line 6, in inner
                  File "/home/jovyan/.local/lib/python3.6/site-packages/asv/benchmark.py", line 512, in <lambda>
                    func = lambda: self.func(*param)
                  File "/home/jovyan/dev/storage-benchmarks/benchmarks/benchmark_tools.py", line 41, in func_wrapper
                    func(*args, **kwargs)
                  File "/home/jovyan/dev/storage-benchmarks/benchmarks/gcp_kubernetes_write_zarr.py", line 62, in time_synthetic_write
                    self.da.store(self.target.storage_obj, lock=False)
                  File "/home/jovyan/dev/storage-benchmarks/.asv/env/4147cb8bacd14ebb1c15e8248dc80f49/lib/python3.6/site-packages/dask/array/core.py", line 1269, in store
                    r = store([self], [target], **kwargs)
                  File "/home/jovyan/dev/storage-benchmarks/.asv/env/4147cb8bacd14ebb1c15e8248dc80f49/lib/python3.6/site-packages/dask/array/core.py", line 1004, in store
                    result.compute(**kwargs)
                  File "/home/jovyan/dev/storage-benchmarks/.asv/env/4147cb8bacd14ebb1c15e8248dc80f49/lib/python3.6/site-packages/dask/base.py", line 156, in compute
                    (result,) = compute(self, traverse=False, **kwargs)
                  File "/home/jovyan/dev/storage-benchmarks/.asv/env/4147cb8bacd14ebb1c15e8248dc80f49/lib/python3.6/site-packages/dask/base.py", line 402, in compute
                    results = schedule(dsk, keys, **kwargs)
                  File "/home/jovyan/dev/storage-benchmarks/.asv/env/4147cb8bacd14ebb1c15e8248dc80f49/lib/python3.6/site-packages/distributed/client.py", line 2159, in get
                    direct=direct)
                  File "/home/jovyan/dev/storage-benchmarks/.asv/env/4147cb8bacd14ebb1c15e8248dc80f49/lib/python3.6/site-packages/distributed/client.py", line 1562, in gather
                    asynchronous=asynchronous)
                  File "/home/jovyan/dev/storage-benchmarks/.asv/env/4147cb8bacd14ebb1c15e8248dc80f49/lib/python3.6/site-packages/distributed/client.py", line 652, in sync
                    return sync(self.loop, func, *args, **kwargs)
                  File "/home/jovyan/dev/storage-benchmarks/.asv/env/4147cb8bacd14ebb1c15e8248dc80f49/lib/python3.6/site-packages/distributed/utils.py", line 275, in sync
                    six.reraise(*error[0])
                  File "/home/jovyan/dev/storage-benchmarks/.asv/env/4147cb8bacd14ebb1c15e8248dc80f49/lib/python3.6/site-packages/six.py", line 693, in reraise
                    raise value
                  File "/home/jovyan/dev/storage-benchmarks/.asv/env/4147cb8bacd14ebb1c15e8248dc80f49/lib/python3.6/site-packages/distributed/utils.py", line 260, in f
                    result[0] = yield make_coro()
                  File "/home/jovyan/dev/storage-benchmarks/.asv/env/4147cb8bacd14ebb1c15e8248dc80f49/lib/python3.6/site-packages/tornado/gen.py", line 1133, in run
                    value = future.result()
                  File "/home/jovyan/dev/storage-benchmarks/.asv/env/4147cb8bacd14ebb1c15e8248dc80f49/lib/python3.6/site-packages/tornado/gen.py", line 1141, in run
                    yielded = self.gen.throw(*exc_info)
                  File "/home/jovyan/dev/storage-benchmarks/.asv/env/4147cb8bacd14ebb1c15e8248dc80f49/lib/python3.6/site-packages/distributed/client.py", line 1439, in _gather
                    traceback)
                  File "/home/jovyan/dev/storage-benchmarks/.asv/env/4147cb8bacd14ebb1c15e8248dc80f49/lib/python3.6/site-packages/six.py", line 692, in reraise
                    raise value.with_traceback(tb)
                  File "/opt/conda/lib/python3.6/site-packages/dask/array/core.py", line 3030, in store_chunk
                    return load_store_chunk(x, out, index, lock, return_stored, False)
                  File "/opt/conda/lib/python3.6/site-packages/dask/array/core.py", line 3019, in load_store_chunk
                    out[index] = np.asanyarray(x)
                  File "/opt/conda/lib/python3.6/site-packages/zarr/core.py", line 1102, in __setitem__
                    self.set_basic_selection(selection, value, fields=fields)
                  File "/opt/conda/lib/python3.6/site-packages/zarr/core.py", line 1197, in set_basic_selection
                    return self._set_basic_selection_nd(selection, value, fields=fields)
                  File "/opt/conda/lib/python3.6/site-packages/zarr/core.py", line 1488, in _set_basic_selection_nd
                    self._set_selection(indexer, value, fields=fields)
                  File "/opt/conda/lib/python3.6/site-packages/zarr/core.py", line 1536, in _set_selection
                    self._chunk_setitem(chunk_coords, chunk_selection, chunk_value, fields=fields)
                  File "/opt/conda/lib/python3.6/site-packages/zarr/core.py", line 1644, in _chunk_setitem
                    fields=fields)
                  File "/opt/conda/lib/python3.6/site-packages/zarr/core.py", line 1721, in _chunk_setitem_nosync
                    self.chunk_store[ckey] = cdata
                  File "/opt/conda/lib/python3.6/site-packages/gcsfs/mapping.py", line 75, in __setitem__
                    f.write(value)
                  File "<decorator-gen-41>", line 2, in __exit__
                  File "/opt/conda/lib/python3.6/site-packages/gcsfs/core.py", line 51, in _tracemethod
                    return f(self, *args, **kwargs)
                  File "/opt/conda/lib/python3.6/site-packages/gcsfs/core.py", line 1432, in __exit__
                    self.close()
                  File "<decorator-gen-38>", line 2, in close
                  File "/opt/conda/lib/python3.6/site-packages/gcsfs/core.py", line 51, in _tracemethod
                    return f(self, *args, **kwargs)
                  File "/opt/conda/lib/python3.6/site-packages/gcsfs/core.py", line 1396, in close
                    self.flush(force=True)
                  File "<decorator-gen-33>", line 2, in flush
                  File "/opt/conda/lib/python3.6/site-packages/gcsfs/core.py", line 51, in _tracemethod
                    return f(self, *args, **kwargs)
                  File "/opt/conda/lib/python3.6/site-packages/gcsfs/core.py", line 1248, in flush
                    self._upload_chunk(final=force)
                  File "<decorator-gen-34>", line 2, in _upload_chunk
                  File "/opt/conda/lib/python3.6/site-packages/gcsfs/core.py", line 51, in _tracemethod
                    return f(self, *args, **kwargs)
                  File "/opt/conda/lib/python3.6/site-packages/gcsfs/core.py", line 1276, in _upload_chunk
                    validate_response(r, self.location)
                  File "/opt/conda/lib/python3.6/site-packages/gcsfs/core.py", line 164, in validate_response
                    raise RuntimeError(m)
                RuntimeError: b'Service Unavailable'
                
                For parameters: 'GCS', 5, 140, 1
                /home/jovyan/dev/storage-benchmarks/.asv/env/4147cb8bacd14ebb1c15e8248dc80f49/lib/python3.6/importlib/_bootstrap.py:219: RuntimeWarning: numpy.dtype size changed, may indicate binary incompatibility. Expected 96, got 88
                  return f(*args, **kwds)
                
                For parameters: 'GCS', 10, 60, 1
                /home/jovyan/dev/storage-benchmarks/.asv/env/4147cb8bacd14ebb1c15e8248dc80f49/lib/python3.6/importlib/_bootstrap.py:219: RuntimeWarning: numpy.dtype size changed, may indicate binary incompatibility. Expected 96, got 88
                  return f(*args, **kwds)
                
                For parameters: 'GCS', 10, 80, 1
                /home/jovyan/dev/storage-benchmarks/.asv/env/4147cb8bacd14ebb1c15e8248dc80f49/lib/python3.6/importlib/_bootstrap.py:219: RuntimeWarning: numpy.dtype size changed, may indicate binary incompatibility. Expected 96, got 88
                  return f(*args, **kwds)
                
                For parameters: 'GCS', 10, 100, 1
                /home/jovyan/dev/storage-benchmarks/.asv/env/4147cb8bacd14ebb1c15e8248dc80f49/lib/python3.6/importlib/_bootstrap.py:219: RuntimeWarning: numpy.dtype size changed, may indicate binary incompatibility. Expected 96, got 88
                  return f(*args, **kwds)
                
                For parameters: 'GCS', 10, 120, 1
                /home/jovyan/dev/storage-benchmarks/.asv/env/4147cb8bacd14ebb1c15e8248dc80f49/lib/python3.6/importlib/_bootstrap.py:219: RuntimeWarning: numpy.dtype size changed, may indicate binary incompatibility. Expected 96, got 88
                  return f(*args, **kwds)
                
                For parameters: 'GCS', 10, 140, 1
                /home/jovyan/dev/storage-benchmarks/.asv/env/4147cb8bacd14ebb1c15e8248dc80f49/lib/python3.6/importlib/_bootstrap.py:219: RuntimeWarning: numpy.dtype size changed, may indicate binary incompatibility. Expected 96, got 88
                  return f(*args, **kwds)

Previous results JSON found. Moving to backup file
· Creating environments
· Discovering benchmarks
·· Uninstalling from conda-py3.6-dask-distributed-gcsfs-h5netcdf-h5py-lz4-netcdf4-numpy-pip+dask_kubernetes-pip+h5pyd-pyyaml-rasterio-xarray-zarr
·· Building for conda-py3.6-dask-distributed-gcsfs-h5netcdf-h5py-lz4-netcdf4-numpy-pip+dask_kubernetes-pip+h5pyd-pyyaml-rasterio-xarray-zarr
·· Installing into conda-py3.6-dask-distributed-gcsfs-h5netcdf-h5py-lz4-netcdf4-numpy-pip+dask_kubernetes-pip+h5pyd-pyyaml-rasterio-xarray-zarr
· Running 2 total benchmarks (1 commits * 1 environments * 2 benchmarks)
[  0.00%] · For pangeo-storage-benchmarks commit hash 75a490e9:
[  0.00%] ·· Building for conda-py3.6-dask-distributed-gcsfs-h5netcdf-h5py-lz4-netcdf4-numpy-pip+dask_kubernetes-pip+h5pyd-pyyaml-rasterio-xarray-zarr
[  0.00%] ·· Benchmarking conda-py3.6-dask-distributed-gcsfs-h5netcdf-h5py-lz4-netcdf4-numpy-pip+dask_kubernetes-pip+h5pyd-pyyaml-rasterio-xarray-zarr
[ 50.00%] ··· Running gcp_kubernetes_write_zarr.rand_ds_size.track_megabytes             137329.1015625
[ 50.00%] ····· /home/jovyan/dev/storage-benchmarks/.asv/env/4147cb8bacd14ebb1c15e8248dc80f49/lib/python3.6/importlib/_bootstrap.py:219: RuntimeWarning: numpy.dtype size changed, may indicate binary incompatibility. Expected 96, got 88
                return f(*args, **kwds)

[100.00%] ··· Running gcp_kubernetes_write_zarr.synthetic_benchmarks.time_synthetic_write   4/15 failed
[100.00%] ···· 
               ========= ============= ======== ======== ========= ========= =========
               --                                    n_workers / run_num              
               ----------------------- -----------------------------------------------
                backend   z_chunksize   60 / 1   80 / 1   100 / 1   120 / 1   140 / 1 
               ========= ============= ======== ======== ========= ========= =========
                  GCS          1        failed   failed    28.93s    25.73s    failed 
                  GCS          5        46.55s   27.80s    28.71s    23.80s    21.19s 
                  GCS          10       failed   1.04m     48.40s    37.65s    40.04s 
               ========= ============= ======== ======== ========= ========= =========

[100.00%] ····· 
                
                For parameters: 'GCS', 1, 60, 1
                /home/jovyan/dev/storage-benchmarks/.asv/env/4147cb8bacd14ebb1c15e8248dc80f49/lib/python3.6/importlib/_bootstrap.py:219: RuntimeWarning: numpy.dtype size changed, may indicate binary incompatibility. Expected 96, got 88
                  return f(*args, **kwds)
                Traceback (most recent call last):
                  File "/home/jovyan/.local/lib/python3.6/site-packages/asv/benchmark.py", line 795, in <module>
                    commands[mode](args)
                  File "/home/jovyan/.local/lib/python3.6/site-packages/asv/benchmark.py", line 772, in main_run
                    result = benchmark.do_run()
                  File "/home/jovyan/.local/lib/python3.6/site-packages/asv/benchmark.py", line 456, in do_run
                    return self.run(*self._current_params)
                  File "/home/jovyan/.local/lib/python3.6/site-packages/asv/benchmark.py", line 548, in run
                    all_runs.extend(timer.repeat(repeat, number))
                  File "/home/jovyan/dev/storage-benchmarks/.asv/env/4147cb8bacd14ebb1c15e8248dc80f49/lib/python3.6/timeit.py", line 206, in repeat
                    t = self.timeit(number)
                  File "/home/jovyan/dev/storage-benchmarks/.asv/env/4147cb8bacd14ebb1c15e8248dc80f49/lib/python3.6/timeit.py", line 178, in timeit
                    timing = self.inner(it, self.timer)
                  File "<timeit-src>", line 6, in inner
                  File "/home/jovyan/.local/lib/python3.6/site-packages/asv/benchmark.py", line 512, in <lambda>
                    func = lambda: self.func(*param)
                  File "/home/jovyan/dev/storage-benchmarks/benchmarks/benchmark_tools.py", line 41, in func_wrapper
                    func(*args, **kwargs)
                  File "/home/jovyan/dev/storage-benchmarks/benchmarks/gcp_kubernetes_write_zarr.py", line 62, in time_synthetic_write
                    self.da.store(self.target.storage_obj, lock=False)
                  File "/home/jovyan/dev/storage-benchmarks/.asv/env/4147cb8bacd14ebb1c15e8248dc80f49/lib/python3.6/site-packages/dask/array/core.py", line 1269, in store
                    r = store([self], [target], **kwargs)
                  File "/home/jovyan/dev/storage-benchmarks/.asv/env/4147cb8bacd14ebb1c15e8248dc80f49/lib/python3.6/site-packages/dask/array/core.py", line 1004, in store
                    result.compute(**kwargs)
                  File "/home/jovyan/dev/storage-benchmarks/.asv/env/4147cb8bacd14ebb1c15e8248dc80f49/lib/python3.6/site-packages/dask/base.py", line 156, in compute
                    (result,) = compute(self, traverse=False, **kwargs)
                  File "/home/jovyan/dev/storage-benchmarks/.asv/env/4147cb8bacd14ebb1c15e8248dc80f49/lib/python3.6/site-packages/dask/base.py", line 402, in compute
                    results = schedule(dsk, keys, **kwargs)
                  File "/home/jovyan/dev/storage-benchmarks/.asv/env/4147cb8bacd14ebb1c15e8248dc80f49/lib/python3.6/site-packages/distributed/client.py", line 2159, in get
                    direct=direct)
                  File "/home/jovyan/dev/storage-benchmarks/.asv/env/4147cb8bacd14ebb1c15e8248dc80f49/lib/python3.6/site-packages/distributed/client.py", line 1562, in gather
                    asynchronous=asynchronous)
                  File "/home/jovyan/dev/storage-benchmarks/.asv/env/4147cb8bacd14ebb1c15e8248dc80f49/lib/python3.6/site-packages/distributed/client.py", line 652, in sync
                    return sync(self.loop, func, *args, **kwargs)
                  File "/home/jovyan/dev/storage-benchmarks/.asv/env/4147cb8bacd14ebb1c15e8248dc80f49/lib/python3.6/site-packages/distributed/utils.py", line 275, in sync
                    six.reraise(*error[0])
                  File "/home/jovyan/dev/storage-benchmarks/.asv/env/4147cb8bacd14ebb1c15e8248dc80f49/lib/python3.6/site-packages/six.py", line 693, in reraise
                    raise value
                  File "/home/jovyan/dev/storage-benchmarks/.asv/env/4147cb8bacd14ebb1c15e8248dc80f49/lib/python3.6/site-packages/distributed/utils.py", line 260, in f
                    result[0] = yield make_coro()
                  File "/home/jovyan/dev/storage-benchmarks/.asv/env/4147cb8bacd14ebb1c15e8248dc80f49/lib/python3.6/site-packages/tornado/gen.py", line 1133, in run
                    value = future.result()
                  File "/home/jovyan/dev/storage-benchmarks/.asv/env/4147cb8bacd14ebb1c15e8248dc80f49/lib/python3.6/site-packages/tornado/gen.py", line 1141, in run
                    yielded = self.gen.throw(*exc_info)
                  File "/home/jovyan/dev/storage-benchmarks/.asv/env/4147cb8bacd14ebb1c15e8248dc80f49/lib/python3.6/site-packages/distributed/client.py", line 1439, in _gather
                    traceback)
                  File "/home/jovyan/dev/storage-benchmarks/.asv/env/4147cb8bacd14ebb1c15e8248dc80f49/lib/python3.6/site-packages/six.py", line 692, in reraise
                    raise value.with_traceback(tb)
                  File "/opt/conda/lib/python3.6/site-packages/dask/array/core.py", line 3030, in store_chunk
                    return load_store_chunk(x, out, index, lock, return_stored, False)
                  File "/opt/conda/lib/python3.6/site-packages/dask/array/core.py", line 3019, in load_store_chunk
                    out[index] = np.asanyarray(x)
                  File "/opt/conda/lib/python3.6/site-packages/zarr/core.py", line 1102, in __setitem__
                    self.set_basic_selection(selection, value, fields=fields)
                  File "/opt/conda/lib/python3.6/site-packages/zarr/core.py", line 1197, in set_basic_selection
                    return self._set_basic_selection_nd(selection, value, fields=fields)
                  File "/opt/conda/lib/python3.6/site-packages/zarr/core.py", line 1488, in _set_basic_selection_nd
                    self._set_selection(indexer, value, fields=fields)
                  File "/opt/conda/lib/python3.6/site-packages/zarr/core.py", line 1536, in _set_selection
                    self._chunk_setitem(chunk_coords, chunk_selection, chunk_value, fields=fields)
                  File "/opt/conda/lib/python3.6/site-packages/zarr/core.py", line 1644, in _chunk_setitem
                    fields=fields)
                  File "/opt/conda/lib/python3.6/site-packages/zarr/core.py", line 1721, in _chunk_setitem_nosync
                    self.chunk_store[ckey] = cdata
                  File "/opt/conda/lib/python3.6/site-packages/gcsfs/mapping.py", line 75, in __setitem__
                    f.write(value)
                  File "/opt/conda/lib/python3.6/site-packages/gcsfs/core.py", line 1198, in write
                    self.flush()
                  File "<decorator-gen-33>", line 2, in flush
                  File "/opt/conda/lib/python3.6/site-packages/gcsfs/core.py", line 51, in _tracemethod
                    return f(self, *args, **kwargs)
                  File "/opt/conda/lib/python3.6/site-packages/gcsfs/core.py", line 1248, in flush
                    self._upload_chunk(final=force)
                  File "<decorator-gen-34>", line 2, in _upload_chunk
                  File "/opt/conda/lib/python3.6/site-packages/gcsfs/core.py", line 51, in _tracemethod
                    return f(self, *args, **kwargs)
                  File "/opt/conda/lib/python3.6/site-packages/gcsfs/core.py", line 1275, in _upload_chunk
                    headers=head, data=data)
                  File "/opt/conda/lib/python3.6/site-packages/requests/sessions.py", line 535, in post
                    r"""Sends a HEAD request. Returns :class:`Response` object.
                  File "/opt/conda/lib/python3.6/site-packages/google/auth/transport/requests.py", line 201, in request
                    method, url, data=data, headers=request_headers, **kwargs)
                  File "/opt/conda/lib/python3.6/site-packages/requests/sessions.py", line 488, in request
                    json=json,
                  File "/opt/conda/lib/python3.6/site-packages/requests/sessions.py", line 609, in send
                    hooks = request.hooks
                  File "/opt/conda/lib/python3.6/site-packages/requests/adapters.py", line 423, in send
                    elif isinstance(timeout, TimeoutSauce):
                  File "/opt/conda/lib/python3.6/site-packages/requests/packages/urllib3/connectionpool.py", line 594, in urlopen
                  File "/opt/conda/lib/python3.6/site-packages/requests/packages/urllib3/connectionpool.py", line 361, in _make_request
                  File "/opt/conda/lib/python3.6/http/client.py", line 1239, in request
                    self._send_request(method, url, body, headers, encode_chunked)
                  File "/opt/conda/lib/python3.6/http/client.py", line 1285, in _send_request
                    self.endheaders(body, encode_chunked=encode_chunked)
                  File "/opt/conda/lib/python3.6/http/client.py", line 1234, in endheaders
                    self._send_output(message_body, encode_chunked=encode_chunked)
                  File "/opt/conda/lib/python3.6/http/client.py", line 1065, in _send_output
                    self.send(chunk)
                  File "/opt/conda/lib/python3.6/http/client.py", line 986, in send
                    self.sock.sendall(data)
                  File "/opt/conda/lib/python3.6/site-packages/requests/packages/urllib3/contrib/pyopenssl.py", line 292, in sendall
                  File "/opt/conda/lib/python3.6/site-packages/requests/packages/urllib3/contrib/pyopenssl.py", line 281, in _send_until_done
                  File "/opt/conda/lib/python3.6/site-packages/OpenSSL/SSL.py", line 1256, in send
                
                  File "/opt/conda/lib/python3.6/site-packages/OpenSSL/SSL.py", line 1166, in _raise_ssl_error
                
                OpenSSL.SSL.SysCallError: (32, 'EPIPE')
                
                For parameters: 'GCS', 1, 80, 1
                /home/jovyan/dev/storage-benchmarks/.asv/env/4147cb8bacd14ebb1c15e8248dc80f49/lib/python3.6/importlib/_bootstrap.py:219: RuntimeWarning: numpy.dtype size changed, may indicate binary incompatibility. Expected 96, got 88
                  return f(*args, **kwds)
                Traceback (most recent call last):
                  File "/home/jovyan/.local/lib/python3.6/site-packages/asv/benchmark.py", line 795, in <module>
                    commands[mode](args)
                  File "/home/jovyan/.local/lib/python3.6/site-packages/asv/benchmark.py", line 772, in main_run
                    result = benchmark.do_run()
                  File "/home/jovyan/.local/lib/python3.6/site-packages/asv/benchmark.py", line 456, in do_run
                    return self.run(*self._current_params)
                  File "/home/jovyan/.local/lib/python3.6/site-packages/asv/benchmark.py", line 548, in run
                    all_runs.extend(timer.repeat(repeat, number))
                  File "/home/jovyan/dev/storage-benchmarks/.asv/env/4147cb8bacd14ebb1c15e8248dc80f49/lib/python3.6/timeit.py", line 206, in repeat
                    t = self.timeit(number)
                  File "/home/jovyan/dev/storage-benchmarks/.asv/env/4147cb8bacd14ebb1c15e8248dc80f49/lib/python3.6/timeit.py", line 178, in timeit
                    timing = self.inner(it, self.timer)
                  File "<timeit-src>", line 6, in inner
                  File "/home/jovyan/.local/lib/python3.6/site-packages/asv/benchmark.py", line 512, in <lambda>
                    func = lambda: self.func(*param)
                  File "/home/jovyan/dev/storage-benchmarks/benchmarks/benchmark_tools.py", line 41, in func_wrapper
                    func(*args, **kwargs)
                  File "/home/jovyan/dev/storage-benchmarks/benchmarks/gcp_kubernetes_write_zarr.py", line 62, in time_synthetic_write
                    self.da.store(self.target.storage_obj, lock=False)
                  File "/home/jovyan/dev/storage-benchmarks/.asv/env/4147cb8bacd14ebb1c15e8248dc80f49/lib/python3.6/site-packages/dask/array/core.py", line 1269, in store
                    r = store([self], [target], **kwargs)
                  File "/home/jovyan/dev/storage-benchmarks/.asv/env/4147cb8bacd14ebb1c15e8248dc80f49/lib/python3.6/site-packages/dask/array/core.py", line 1004, in store
                    result.compute(**kwargs)
                  File "/home/jovyan/dev/storage-benchmarks/.asv/env/4147cb8bacd14ebb1c15e8248dc80f49/lib/python3.6/site-packages/dask/base.py", line 156, in compute
                    (result,) = compute(self, traverse=False, **kwargs)
                  File "/home/jovyan/dev/storage-benchmarks/.asv/env/4147cb8bacd14ebb1c15e8248dc80f49/lib/python3.6/site-packages/dask/base.py", line 402, in compute
                    results = schedule(dsk, keys, **kwargs)
                  File "/home/jovyan/dev/storage-benchmarks/.asv/env/4147cb8bacd14ebb1c15e8248dc80f49/lib/python3.6/site-packages/distributed/client.py", line 2159, in get
                    direct=direct)
                  File "/home/jovyan/dev/storage-benchmarks/.asv/env/4147cb8bacd14ebb1c15e8248dc80f49/lib/python3.6/site-packages/distributed/client.py", line 1562, in gather
                    asynchronous=asynchronous)
                  File "/home/jovyan/dev/storage-benchmarks/.asv/env/4147cb8bacd14ebb1c15e8248dc80f49/lib/python3.6/site-packages/distributed/client.py", line 652, in sync
                    return sync(self.loop, func, *args, **kwargs)
                  File "/home/jovyan/dev/storage-benchmarks/.asv/env/4147cb8bacd14ebb1c15e8248dc80f49/lib/python3.6/site-packages/distributed/utils.py", line 275, in sync
                    six.reraise(*error[0])
                  File "/home/jovyan/dev/storage-benchmarks/.asv/env/4147cb8bacd14ebb1c15e8248dc80f49/lib/python3.6/site-packages/six.py", line 693, in reraise
                    raise value
                  File "/home/jovyan/dev/storage-benchmarks/.asv/env/4147cb8bacd14ebb1c15e8248dc80f49/lib/python3.6/site-packages/distributed/utils.py", line 260, in f
                    result[0] = yield make_coro()
                  File "/home/jovyan/dev/storage-benchmarks/.asv/env/4147cb8bacd14ebb1c15e8248dc80f49/lib/python3.6/site-packages/tornado/gen.py", line 1133, in run
                    value = future.result()
                  File "/home/jovyan/dev/storage-benchmarks/.asv/env/4147cb8bacd14ebb1c15e8248dc80f49/lib/python3.6/site-packages/tornado/gen.py", line 1141, in run
                    yielded = self.gen.throw(*exc_info)
                  File "/home/jovyan/dev/storage-benchmarks/.asv/env/4147cb8bacd14ebb1c15e8248dc80f49/lib/python3.6/site-packages/distributed/client.py", line 1439, in _gather
                    traceback)
                  File "/home/jovyan/dev/storage-benchmarks/.asv/env/4147cb8bacd14ebb1c15e8248dc80f49/lib/python3.6/site-packages/six.py", line 692, in reraise
                    raise value.with_traceback(tb)
                  File "/opt/conda/lib/python3.6/site-packages/dask/array/core.py", line 3030, in store_chunk
                    return load_store_chunk(x, out, index, lock, return_stored, False)
                  File "/opt/conda/lib/python3.6/site-packages/dask/array/core.py", line 3019, in load_store_chunk
                    out[index] = np.asanyarray(x)
                  File "/opt/conda/lib/python3.6/site-packages/zarr/core.py", line 1102, in __setitem__
                    self.set_basic_selection(selection, value, fields=fields)
                  File "/opt/conda/lib/python3.6/site-packages/zarr/core.py", line 1197, in set_basic_selection
                    return self._set_basic_selection_nd(selection, value, fields=fields)
                  File "/opt/conda/lib/python3.6/site-packages/zarr/core.py", line 1488, in _set_basic_selection_nd
                    self._set_selection(indexer, value, fields=fields)
                  File "/opt/conda/lib/python3.6/site-packages/zarr/core.py", line 1536, in _set_selection
                    self._chunk_setitem(chunk_coords, chunk_selection, chunk_value, fields=fields)
                  File "/opt/conda/lib/python3.6/site-packages/zarr/core.py", line 1644, in _chunk_setitem
                    fields=fields)
                  File "/opt/conda/lib/python3.6/site-packages/zarr/core.py", line 1721, in _chunk_setitem_nosync
                    self.chunk_store[ckey] = cdata
                  File "/opt/conda/lib/python3.6/site-packages/gcsfs/mapping.py", line 75, in __setitem__
                    f.write(value)
                  File "/opt/conda/lib/python3.6/site-packages/gcsfs/core.py", line 1198, in write
                    self.flush()
                  File "<decorator-gen-33>", line 2, in flush
                  File "/opt/conda/lib/python3.6/site-packages/gcsfs/core.py", line 51, in _tracemethod
                    return f(self, *args, **kwargs)
                  File "/opt/conda/lib/python3.6/site-packages/gcsfs/core.py", line 1244, in flush
                    self._initiate_upload()
                  File "<decorator-gen-35>", line 2, in _initiate_upload
                  File "/opt/conda/lib/python3.6/site-packages/gcsfs/core.py", line 51, in _tracemethod
                    return f(self, *args, **kwargs)
                  File "/opt/conda/lib/python3.6/site-packages/gcsfs/core.py", line 1309, in _initiate_upload
                    self.location = r.headers['Location']
                  File "/opt/conda/lib/python3.6/site-packages/requests/structures.py", line 54, in __getitem__
                    return self._store[key.lower()][1]
                KeyError: 'location'
                
                For parameters: 'GCS', 1, 100, 1
                /home/jovyan/dev/storage-benchmarks/.asv/env/4147cb8bacd14ebb1c15e8248dc80f49/lib/python3.6/importlib/_bootstrap.py:219: RuntimeWarning: numpy.dtype size changed, may indicate binary incompatibility. Expected 96, got 88
                  return f(*args, **kwds)
                
                For parameters: 'GCS', 1, 120, 1
                /home/jovyan/dev/storage-benchmarks/.asv/env/4147cb8bacd14ebb1c15e8248dc80f49/lib/python3.6/importlib/_bootstrap.py:219: RuntimeWarning: numpy.dtype size changed, may indicate binary incompatibility. Expected 96, got 88
                  return f(*args, **kwds)
                
                For parameters: 'GCS', 1, 140, 1
                /home/jovyan/dev/storage-benchmarks/.asv/env/4147cb8bacd14ebb1c15e8248dc80f49/lib/python3.6/importlib/_bootstrap.py:219: RuntimeWarning: numpy.dtype size changed, may indicate binary incompatibility. Expected 96, got 88
                  return f(*args, **kwds)
                Traceback (most recent call last):
                  File "/home/jovyan/.local/lib/python3.6/site-packages/asv/benchmark.py", line 795, in <module>
                    commands[mode](args)
                  File "/home/jovyan/.local/lib/python3.6/site-packages/asv/benchmark.py", line 772, in main_run
                    result = benchmark.do_run()
                  File "/home/jovyan/.local/lib/python3.6/site-packages/asv/benchmark.py", line 456, in do_run
                    return self.run(*self._current_params)
                  File "/home/jovyan/.local/lib/python3.6/site-packages/asv/benchmark.py", line 548, in run
                    all_runs.extend(timer.repeat(repeat, number))
                  File "/home/jovyan/dev/storage-benchmarks/.asv/env/4147cb8bacd14ebb1c15e8248dc80f49/lib/python3.6/timeit.py", line 206, in repeat
                    t = self.timeit(number)
                  File "/home/jovyan/dev/storage-benchmarks/.asv/env/4147cb8bacd14ebb1c15e8248dc80f49/lib/python3.6/timeit.py", line 178, in timeit
                    timing = self.inner(it, self.timer)
                  File "<timeit-src>", line 6, in inner
                  File "/home/jovyan/.local/lib/python3.6/site-packages/asv/benchmark.py", line 512, in <lambda>
                    func = lambda: self.func(*param)
                  File "/home/jovyan/dev/storage-benchmarks/benchmarks/benchmark_tools.py", line 41, in func_wrapper
                    func(*args, **kwargs)
                  File "/home/jovyan/dev/storage-benchmarks/benchmarks/gcp_kubernetes_write_zarr.py", line 62, in time_synthetic_write
                    self.da.store(self.target.storage_obj, lock=False)
                  File "/home/jovyan/dev/storage-benchmarks/.asv/env/4147cb8bacd14ebb1c15e8248dc80f49/lib/python3.6/site-packages/dask/array/core.py", line 1269, in store
                    r = store([self], [target], **kwargs)
                  File "/home/jovyan/dev/storage-benchmarks/.asv/env/4147cb8bacd14ebb1c15e8248dc80f49/lib/python3.6/site-packages/dask/array/core.py", line 1004, in store
                    result.compute(**kwargs)
                  File "/home/jovyan/dev/storage-benchmarks/.asv/env/4147cb8bacd14ebb1c15e8248dc80f49/lib/python3.6/site-packages/dask/base.py", line 156, in compute
                    (result,) = compute(self, traverse=False, **kwargs)
                  File "/home/jovyan/dev/storage-benchmarks/.asv/env/4147cb8bacd14ebb1c15e8248dc80f49/lib/python3.6/site-packages/dask/base.py", line 402, in compute
                    results = schedule(dsk, keys, **kwargs)
                  File "/home/jovyan/dev/storage-benchmarks/.asv/env/4147cb8bacd14ebb1c15e8248dc80f49/lib/python3.6/site-packages/distributed/client.py", line 2159, in get
                    direct=direct)
                  File "/home/jovyan/dev/storage-benchmarks/.asv/env/4147cb8bacd14ebb1c15e8248dc80f49/lib/python3.6/site-packages/distributed/client.py", line 1562, in gather
                    asynchronous=asynchronous)
                  File "/home/jovyan/dev/storage-benchmarks/.asv/env/4147cb8bacd14ebb1c15e8248dc80f49/lib/python3.6/site-packages/distributed/client.py", line 652, in sync
                    return sync(self.loop, func, *args, **kwargs)
                  File "/home/jovyan/dev/storage-benchmarks/.asv/env/4147cb8bacd14ebb1c15e8248dc80f49/lib/python3.6/site-packages/distributed/utils.py", line 275, in sync
                    six.reraise(*error[0])
                  File "/home/jovyan/dev/storage-benchmarks/.asv/env/4147cb8bacd14ebb1c15e8248dc80f49/lib/python3.6/site-packages/six.py", line 693, in reraise
                    raise value
                  File "/home/jovyan/dev/storage-benchmarks/.asv/env/4147cb8bacd14ebb1c15e8248dc80f49/lib/python3.6/site-packages/distributed/utils.py", line 260, in f
                    result[0] = yield make_coro()
                  File "/home/jovyan/dev/storage-benchmarks/.asv/env/4147cb8bacd14ebb1c15e8248dc80f49/lib/python3.6/site-packages/tornado/gen.py", line 1133, in run
                    value = future.result()
                  File "/home/jovyan/dev/storage-benchmarks/.asv/env/4147cb8bacd14ebb1c15e8248dc80f49/lib/python3.6/site-packages/tornado/gen.py", line 1141, in run
                    yielded = self.gen.throw(*exc_info)
                  File "/home/jovyan/dev/storage-benchmarks/.asv/env/4147cb8bacd14ebb1c15e8248dc80f49/lib/python3.6/site-packages/distributed/client.py", line 1439, in _gather
                    traceback)
                  File "/home/jovyan/dev/storage-benchmarks/.asv/env/4147cb8bacd14ebb1c15e8248dc80f49/lib/python3.6/site-packages/six.py", line 692, in reraise
                    raise value.with_traceback(tb)
                  File "/opt/conda/lib/python3.6/site-packages/dask/array/core.py", line 3030, in store_chunk
                    return load_store_chunk(x, out, index, lock, return_stored, False)
                  File "/opt/conda/lib/python3.6/site-packages/dask/array/core.py", line 3019, in load_store_chunk
                    out[index] = np.asanyarray(x)
                  File "/opt/conda/lib/python3.6/site-packages/zarr/core.py", line 1102, in __setitem__
                    self.set_basic_selection(selection, value, fields=fields)
                  File "/opt/conda/lib/python3.6/site-packages/zarr/core.py", line 1197, in set_basic_selection
                    return self._set_basic_selection_nd(selection, value, fields=fields)
                  File "/opt/conda/lib/python3.6/site-packages/zarr/core.py", line 1488, in _set_basic_selection_nd
                    self._set_selection(indexer, value, fields=fields)
                  File "/opt/conda/lib/python3.6/site-packages/zarr/core.py", line 1536, in _set_selection
                    self._chunk_setitem(chunk_coords, chunk_selection, chunk_value, fields=fields)
                  File "/opt/conda/lib/python3.6/site-packages/zarr/core.py", line 1644, in _chunk_setitem
                    fields=fields)
                  File "/opt/conda/lib/python3.6/site-packages/zarr/core.py", line 1721, in _chunk_setitem_nosync
                    self.chunk_store[ckey] = cdata
                  File "/opt/conda/lib/python3.6/site-packages/gcsfs/mapping.py", line 75, in __setitem__
                    f.write(value)
                  File "/opt/conda/lib/python3.6/site-packages/gcsfs/core.py", line 1198, in write
                    self.flush()
                  File "<decorator-gen-33>", line 2, in flush
                  File "/opt/conda/lib/python3.6/site-packages/gcsfs/core.py", line 51, in _tracemethod
                    return f(self, *args, **kwargs)
                  File "/opt/conda/lib/python3.6/site-packages/gcsfs/core.py", line 1244, in flush
                    self._initiate_upload()
                  File "<decorator-gen-35>", line 2, in _initiate_upload
                  File "/opt/conda/lib/python3.6/site-packages/gcsfs/core.py", line 51, in _tracemethod
                    return f(self, *args, **kwargs)
                  File "/opt/conda/lib/python3.6/site-packages/gcsfs/core.py", line 1309, in _initiate_upload
                    self.location = r.headers['Location']
                  File "/opt/conda/lib/python3.6/site-packages/requests/structures.py", line 54, in __getitem__
                    return self._store[key.lower()][1]
                KeyError: 'location'
                
                For parameters: 'GCS', 5, 60, 1
                /home/jovyan/dev/storage-benchmarks/.asv/env/4147cb8bacd14ebb1c15e8248dc80f49/lib/python3.6/importlib/_bootstrap.py:219: RuntimeWarning: numpy.dtype size changed, may indicate binary incompatibility. Expected 96, got 88
                  return f(*args, **kwds)
                
                For parameters: 'GCS', 5, 80, 1
                /home/jovyan/dev/storage-benchmarks/.asv/env/4147cb8bacd14ebb1c15e8248dc80f49/lib/python3.6/importlib/_bootstrap.py:219: RuntimeWarning: numpy.dtype size changed, may indicate binary incompatibility. Expected 96, got 88
                  return f(*args, **kwds)
                
                For parameters: 'GCS', 5, 100, 1
                /home/jovyan/dev/storage-benchmarks/.asv/env/4147cb8bacd14ebb1c15e8248dc80f49/lib/python3.6/importlib/_bootstrap.py:219: RuntimeWarning: numpy.dtype size changed, may indicate binary incompatibility. Expected 96, got 88
                  return f(*args, **kwds)
                
                For parameters: 'GCS', 5, 120, 1
                /home/jovyan/dev/storage-benchmarks/.asv/env/4147cb8bacd14ebb1c15e8248dc80f49/lib/python3.6/importlib/_bootstrap.py:219: RuntimeWarning: numpy.dtype size changed, may indicate binary incompatibility. Expected 96, got 88
                  return f(*args, **kwds)
                
                For parameters: 'GCS', 5, 140, 1
                /home/jovyan/dev/storage-benchmarks/.asv/env/4147cb8bacd14ebb1c15e8248dc80f49/lib/python3.6/importlib/_bootstrap.py:219: RuntimeWarning: numpy.dtype size changed, may indicate binary incompatibility. Expected 96, got 88
                  return f(*args, **kwds)
                
                For parameters: 'GCS', 10, 60, 1
                /home/jovyan/dev/storage-benchmarks/.asv/env/4147cb8bacd14ebb1c15e8248dc80f49/lib/python3.6/importlib/_bootstrap.py:219: RuntimeWarning: numpy.dtype size changed, may indicate binary incompatibility. Expected 96, got 88
                  return f(*args, **kwds)
                Traceback (most recent call last):
                  File "/home/jovyan/.local/lib/python3.6/site-packages/asv/benchmark.py", line 795, in <module>
                    commands[mode](args)
                  File "/home/jovyan/.local/lib/python3.6/site-packages/asv/benchmark.py", line 772, in main_run
                    result = benchmark.do_run()
                  File "/home/jovyan/.local/lib/python3.6/site-packages/asv/benchmark.py", line 456, in do_run
                    return self.run(*self._current_params)
                  File "/home/jovyan/.local/lib/python3.6/site-packages/asv/benchmark.py", line 548, in run
                    all_runs.extend(timer.repeat(repeat, number))
                  File "/home/jovyan/dev/storage-benchmarks/.asv/env/4147cb8bacd14ebb1c15e8248dc80f49/lib/python3.6/timeit.py", line 206, in repeat
                    t = self.timeit(number)
                  File "/home/jovyan/dev/storage-benchmarks/.asv/env/4147cb8bacd14ebb1c15e8248dc80f49/lib/python3.6/timeit.py", line 178, in timeit
                    timing = self.inner(it, self.timer)
                  File "<timeit-src>", line 6, in inner
                  File "/home/jovyan/.local/lib/python3.6/site-packages/asv/benchmark.py", line 512, in <lambda>
                    func = lambda: self.func(*param)
                  File "/home/jovyan/dev/storage-benchmarks/benchmarks/benchmark_tools.py", line 41, in func_wrapper
                    func(*args, **kwargs)
                  File "/home/jovyan/dev/storage-benchmarks/benchmarks/gcp_kubernetes_write_zarr.py", line 62, in time_synthetic_write
                    self.da.store(self.target.storage_obj, lock=False)
                  File "/home/jovyan/dev/storage-benchmarks/.asv/env/4147cb8bacd14ebb1c15e8248dc80f49/lib/python3.6/site-packages/dask/array/core.py", line 1269, in store
                    r = store([self], [target], **kwargs)
                  File "/home/jovyan/dev/storage-benchmarks/.asv/env/4147cb8bacd14ebb1c15e8248dc80f49/lib/python3.6/site-packages/dask/array/core.py", line 1004, in store
                    result.compute(**kwargs)
                  File "/home/jovyan/dev/storage-benchmarks/.asv/env/4147cb8bacd14ebb1c15e8248dc80f49/lib/python3.6/site-packages/dask/base.py", line 156, in compute
                    (result,) = compute(self, traverse=False, **kwargs)
                  File "/home/jovyan/dev/storage-benchmarks/.asv/env/4147cb8bacd14ebb1c15e8248dc80f49/lib/python3.6/site-packages/dask/base.py", line 402, in compute
                    results = schedule(dsk, keys, **kwargs)
                  File "/home/jovyan/dev/storage-benchmarks/.asv/env/4147cb8bacd14ebb1c15e8248dc80f49/lib/python3.6/site-packages/distributed/client.py", line 2159, in get
                    direct=direct)
                  File "/home/jovyan/dev/storage-benchmarks/.asv/env/4147cb8bacd14ebb1c15e8248dc80f49/lib/python3.6/site-packages/distributed/client.py", line 1562, in gather
                    asynchronous=asynchronous)
                  File "/home/jovyan/dev/storage-benchmarks/.asv/env/4147cb8bacd14ebb1c15e8248dc80f49/lib/python3.6/site-packages/distributed/client.py", line 652, in sync
                    return sync(self.loop, func, *args, **kwargs)
                  File "/home/jovyan/dev/storage-benchmarks/.asv/env/4147cb8bacd14ebb1c15e8248dc80f49/lib/python3.6/site-packages/distributed/utils.py", line 275, in sync
                    six.reraise(*error[0])
                  File "/home/jovyan/dev/storage-benchmarks/.asv/env/4147cb8bacd14ebb1c15e8248dc80f49/lib/python3.6/site-packages/six.py", line 693, in reraise
                    raise value
                  File "/home/jovyan/dev/storage-benchmarks/.asv/env/4147cb8bacd14ebb1c15e8248dc80f49/lib/python3.6/site-packages/distributed/utils.py", line 260, in f
                    result[0] = yield make_coro()
                  File "/home/jovyan/dev/storage-benchmarks/.asv/env/4147cb8bacd14ebb1c15e8248dc80f49/lib/python3.6/site-packages/tornado/gen.py", line 1133, in run
                    value = future.result()
                  File "/home/jovyan/dev/storage-benchmarks/.asv/env/4147cb8bacd14ebb1c15e8248dc80f49/lib/python3.6/site-packages/tornado/gen.py", line 1141, in run
                    yielded = self.gen.throw(*exc_info)
                  File "/home/jovyan/dev/storage-benchmarks/.asv/env/4147cb8bacd14ebb1c15e8248dc80f49/lib/python3.6/site-packages/distributed/client.py", line 1439, in _gather
                    traceback)
                  File "/home/jovyan/dev/storage-benchmarks/.asv/env/4147cb8bacd14ebb1c15e8248dc80f49/lib/python3.6/site-packages/six.py", line 693, in reraise
                    raise value
                distributed.scheduler.KilledWorker: ("('normal-4e3bc0d6ee664b7d0d0ac81ee0a6efd4', 0, 0, 190)", 'tcp://10.24.71.21:42276')
                
                For parameters: 'GCS', 10, 80, 1
                /home/jovyan/dev/storage-benchmarks/.asv/env/4147cb8bacd14ebb1c15e8248dc80f49/lib/python3.6/importlib/_bootstrap.py:219: RuntimeWarning: numpy.dtype size changed, may indicate binary incompatibility. Expected 96, got 88
                  return f(*args, **kwds)
                
                For parameters: 'GCS', 10, 100, 1
                /home/jovyan/dev/storage-benchmarks/.asv/env/4147cb8bacd14ebb1c15e8248dc80f49/lib/python3.6/importlib/_bootstrap.py:219: RuntimeWarning: numpy.dtype size changed, may indicate binary incompatibility. Expected 96, got 88
                  return f(*args, **kwds)
                
                For parameters: 'GCS', 10, 120, 1
                /home/jovyan/dev/storage-benchmarks/.asv/env/4147cb8bacd14ebb1c15e8248dc80f49/lib/python3.6/importlib/_bootstrap.py:219: RuntimeWarning: numpy.dtype size changed, may indicate binary incompatibility. Expected 96, got 88
                  return f(*args, **kwds)
                
                For parameters: 'GCS', 10, 140, 1
                /home/jovyan/dev/storage-benchmarks/.asv/env/4147cb8bacd14ebb1c15e8248dc80f49/lib/python3.6/importlib/_bootstrap.py:219: RuntimeWarning: numpy.dtype size changed, may indicate binary incompatibility. Expected 96, got 88
                  return f(*args, **kwds)

Previous results JSON found. Moving to backup file
